### ISSUE TITLE
ripping fails due to bad character range

### ISSUE THREAD

**Issue by dietrmat** [whipper-team/whipper#556]
Hi there, unfortunately ripping for one cd fails due to hitting a bad character range at the end of the rip:
```
/usr/lib/python3.10/site-packages/pkg_resources/__init__.py:116: PkgResourcesDeprecationWarning: 2.0.5-build-libtorrent-rasterbar-src-libtorrent-rasterbar-2.0.5-bindings-python is an invalid version and will not be supported in a future release
  warnings.warn(
CDDB disc id: 7c0bb91e
MusicBrainz disc id wvY7nb0LLpgCbg6nGPbxMx7o46Y-
MusicBrainz lookup URL https://musicbrainz.org/cdtoc/attach?toc=1+30+225280+150+7427+15014+22532+29874+37607+45532+51825+60211+67493+74469+83099+90425+97607+105382+112590+119915+127316+135004+142577+148370+157803+164885+172787+180620+187683+194674+202763+210089+217544&tracks=30&id=wvY7nb0LLpgCbg6nGPbxMx7o46Y-
Disc duration: 00:50:01.733, 30 audio tracks

Matching releases:

Artist  : Spirit - Wild und frei
Title   : Spirit - Wild und frei - Doppel-Box (Folge 11 + 12) - Das Original-Hörspiel zur TV-Serie (Disc 2 of 2): Folge 12: Tante Cora muss bleiben / Lucky und der Zirkus (Teil 1 & 2) [Das Original-Hörspiel zur TV-Serie]
Duration: 00:50:01.723
URL     : https://musicbrainz.org/release/5cf6bdbc-eaf6-4348-b172-225eb2f557f8
Release : 5cf6bdbc-eaf6-4348-b172-225eb2f557f8
Type    : Other
Barcode : 4029759144687
Country : Germany
Cat no  : 0214468KID

Track 1 finished, found 5 Q sub-channels with CRC errors
Track 2 finished, found 2 Q sub-channels with CRC errors
Track 3 finished, found 5 Q sub-channels with CRC errors
Track 4 finished, found 5 Q sub-channels with CRC errors
Track 5 finished, found 1 Q sub-channels with CRC errors
Track 6 finished, found 3 Q sub-channels with CRC errors
Track 8 finished, found 1 Q sub-channels with CRC errors
Track 9 finished, found 3 Q sub-channels with CRC errors
Track 10 finished, found 7 Q sub-channels with CRC errors
Track 11 finished, found 14 Q sub-channels with CRC errors
Track 12 finished, found 1 Q sub-channels with CRC errors
Track 13 finished, found 6 Q sub-channels with CRC errors
Track 14 finished, found 14 Q sub-channels with CRC errors
Track 15 finished, found 2 Q sub-channels with CRC errors
Track 16 finished, found 2 Q sub-channels with CRC errors
Track 17 finished, found 2 Q sub-channels with CRC errors
Track 18 finished, found 1 Q sub-channels with CRC errors
Track 19 finished, found 2 Q sub-channels with CRC errors
Track 20 finished, found 1 Q sub-channels with CRC errors
Track 21 finished, found 4 Q sub-channels with CRC errors
Track 22 finished, found 2 Q sub-channels with CRC errors
Track 23 finished, found 2 Q sub-channels with CRC errors
Track 24 finished, found 6 Q sub-channels with CRC errors
Track 25 finished, found 3 Q sub-channels with CRC errors
Track 26 finished, found 1 Q sub-channels with CRC errors
Track 27 finished, found 2 Q sub-channels with CRC errors
Track 28 finished, found 1 Q sub-channels with CRC errors
Track 29 finished, found 2 Q sub-channels with CRC errors
Track 30 finished, found 7 Q sub-channels with CRC errors
Traceback (most recent call last):
  File "/usr/bin/whipper", line 33, in <module>
    sys.exit(load_entry_point('whipper==0.10.0', 'console_scripts', 'whipper')())
  File "/usr/lib/python3.10/site-packages/whipper/command/main.py", line 56, in main
    ret = cmd.do()
  File "/usr/lib/python3.10/site-packages/whipper/command/basecommand.py", line 141, in do
    return self.cmd.do()
  File "/usr/lib/python3.10/site-packages/whipper/command/basecommand.py", line 141, in do
    return self.cmd.do()
  File "/usr/lib/python3.10/site-packages/whipper/command/cd.py", line 203, in do
    ret = self.doCommand()
  File "/usr/lib/python3.10/site-packages/whipper/command/cd.py", line 381, in doCommand
    logs = glob.glob(os.path.join(dirname, '*.log'))
  File "/usr/lib/python3.10/glob.py", line 24, in glob
    return list(iglob(pathname, root_dir=root_dir, dir_fd=dir_fd, recursive=recursive))
  File "/usr/lib/python3.10/glob.py", line 85, in _iglob
    for dirname in dirs:
  File "/usr/lib/python3.10/glob.py", line 86, in _iglob
    for name in glob_in_dir(_join(root_dir, dirname), basename, dir_fd, dironly):
  File "/usr/lib/python3.10/glob.py", line 97, in _glob1
    return fnmatch.filter(names, pattern)
  File "/usr/lib/python3.10/fnmatch.py", line 58, in filter
    match = _compile_pattern(pat)
  File "/usr/lib/python3.10/fnmatch.py", line 52, in _compile_pattern
    return re.compile(res).match
  File "/usr/lib/python3.10/re.py", line 251, in compile
    return _compile(pattern, flags)
  File "/usr/lib/python3.10/re.py", line 303, in _compile
    p = sre_compile.compile(pattern, flags)
  File "/usr/lib/python3.10/sre_compile.py", line 764, in compile
    p = sre_parse.parse(p, flags)
  File "/usr/lib/python3.10/sre_parse.py", line 948, in parse
    p = _parse_sub(source, state, flags & SRE_FLAG_VERBOSE, 0)
  File "/usr/lib/python3.10/sre_parse.py", line 443, in _parse_sub
    itemsappend(_parse(source, state, verbose, nested + 1,
  File "/usr/lib/python3.10/sre_parse.py", line 834, in _parse
    p = _parse_sub(source, state, sub_verbose, nested + 1)
  File "/usr/lib/python3.10/sre_parse.py", line 443, in _parse_sub
    itemsappend(_parse(source, state, verbose, nested + 1,
  File "/usr/lib/python3.10/sre_parse.py", line 598, in _parse
    raise source.error(msg, len(this) + 1 + len(that))
re.error: bad character range l-H at position 272
```
Debug log is attached.
[whipper.log](https://github.com/whipper-team/whipper/files/8459813/whipper.log)

**Comment by Jormangeud** [comment]
This has been fixed in #548

**Comment by JoeLametta** [comment]
This issue seems to be a duplicate of #547 which, as written by Jormangeud above, has been already fixed with #548. Unfortunately the bugfix is not included in whipper [v0.10.0](https://github.com/whipper-team/whipper/releases/tag/v0.10.0) but will be available in the next version.

**Comment by thomas-mc-work** [comment]
May I kindly ask for a new release here? Beyond that I guess the project is stable enough to switch to more frequent releases. That'll bring improvements faster to the users and might also lower the entry barrier for contributors (faster path to success).
